docs/src/main/sphinx/functions/aggregate.md

```diff
@@ -198,6 +204,21 @@ csv_value
 'a,b,c'
 ```
 
+The following example casts a `value` to `varchar`:
```

Comment:
what do you mean by "a `value`" ?

Maybe rather use "the `v` column" and replace "value" with "v"